translations/chi/get5.phrases.txt

```diff
@@ -122,7 +122,7 @@
     }
     "ForceReadyInfoMessage"
     {
-        "chi"            "如果您有不到{1}名玩家，您可以输入{2}来强制准备您的队伍。"
+        "chi"            "您可以输入{1}来强制准备您的队伍。"
```

Comment:
According to Google, this variable was in the wrong place, so I've best-efforted it to be correct.